feat: export class MaxPartSizeExceededError from `adapters/node-http/content-type/form-data/uploadHandler`

### Describe the feature you'd like to request

The `experimental_parseMultipartFormData` function has a default `maxPartSize` of 3mb, which, when surpassed, the function throws an instance of `MaxPartSizeExceededError`.

I have tried to import the class in my code from `@trpc/server/dist/adapters/node-http/content-type/form-data/uploadHandler`, but I get an error from the bundler saying `Module not found: Package path ./dist/adapters/node-http/content-type/form-data/uploadHandler is not exported from package`.

I would like to be able to handle that error and throw a `TRPCError` that I can then handle on the client.

### Describe the solution you'd like to see

If possible, it would be great to have the class exposed for error handling.

### Describe alternate solutions

-

### Additional information

_No response_

### 👨‍👧‍👦 Contributing

- [X] 🙋‍♂️ Yes, I'd be down to file a PR implementing this feature!

<!-- POLAR PLEDGE BADGE START -->
## Funding
* You can sponsor this specific effort via a [Polar.sh](https://polar.sh) pledge below
* We receive the pledge once the issue is completed & verified

<a href="https://polar.sh/trpc/trpc/issues/4867">
<picture>
  <source media="(prefers-color-scheme: dark)" srcset="https://polar.sh/api/github/trpc/trpc/issues/4867/pledge.svg?darkmode=1">
  <img alt="Fund with Polar" src="https://polar.sh/api/github/trpc/trpc/issues/4867/pledge.svg">
</picture>
</a>
<!-- POLAR PLEDGE BADGE END -->

--- a/packages/server/src/adapters/node-http/content-type/form-data/index.ts
+++ b/packages/server/src/adapters/node-http/content-type/form-data/index.ts
@@ -100,6 +100,9 @@ export const nodeHTTPFormDataContentTypeHandler =
 export { parseMultipartFormData as experimental_parseMultipartFormData };
 export { createMemoryUploadHandler as experimental_createMemoryUploadHandler } from './memoryUploadHandler';
 export { createFileUploadHandler as experimental_createFileUploadHandler } from './fileUploadHandler';
-export { composeUploadHandlers as experimental_composeUploadHandlers } from './uploadHandler';
+export {
+  composeUploadHandlers as experimental_composeUploadHandlers,
+  MaxPartSizeExceededError,
+} from './uploadHandler';
 export { type UploadHandler } from './uploadHandler';
 export { isMultipartFormDataRequest as experimental_isMultipartFormDataRequest };

